logrotate.c

```diff
@@ -371,13 +371,16 @@ static int setSecCtx(int fdSrc, const char *src, char **pPrevCtx)
     /* save default security context for restoreSecCtx() */
     if (getfscreatecon_raw(pPrevCtx) < 0) {
         message(MESS_ERROR, "getting default context: %s\n", strerror(errno));
+        freecon(*pPrevCtx);
```

Comment:
I don't think we can free `*pPrevCtx` here, because callers still can use (free or restore from) it if SELinux is in permissve mode.

Reply:
Thank you, removed :+1: